experiments/ClimaEarth/run_amip.jl

```diff
@@ -166,8 +163,31 @@ land_sim_name = "bucket"
 t_end = Float64(time_to_seconds(config_dict["t_end"]))
 t_start = 0.0
 tspan = (t_start, t_end)
-Δt_component = Float64(time_to_seconds(config_dict["dt"]))
 Δt_cpl = Float64(config_dict["dt_cpl"])
+component_dt_names = [:dt_atmos, :dt_land, :dt_ocean, :dt_seaice]
+# if any of the component dt's are not specified, set them all to the overall dt
+if all(key -> !isnothing(config_dict[String(key)]), component_dt_names)
+    haskey(config_dict, "dt") && @warn "Ignoring dt in favor of individual component dt's"
+    for key in component_dt_names
+        component_dt = Float64(time_to_seconds(config_dict[String(key)]))
+        @assert Δt_cpl % component_dt == 0.0 "Coupler dt must be divisible by all component dt's\n dt_cpl = $Δt_cpl\n $key = $component_dt"
+        eval(:($key = $component_dt))
+    end
+    # the Atmos `get_simulation` function expects the atmos config to contain a value dt,
+    # which describes it simulation timestep, so we set it here
+    config_dict["dt"] = config_dict["dt_atmos"]
```

Comment:
It would be better to isolate this workaround to the atmosphere config dict. Could you move this to [get_atmos_config_dict](https://github.com/CliMA/ClimaCoupler.jl/blob/91878a67ec31321f198b5804ad3fe25720c44480/experiments/ClimaEarth/components/atmosphere/climaatmos.jl#L304)? You can use similar logic in there to check if the coupler config dict has `dt` or `dt_atmos`, and choose which to use.

I realized that if `dt_atmos` is specified but another component model dt isn't specified, we could have both `dt` and `dt_atmos` entries in the coupler config, and then it would be repetitive logic to choose which to use in the atmos file. To avoid that, could you also remove any entry in the config dict that gets ignored? I.e. anything that produces a warning in line 183